docs/application/dotnet/guides/nui/nui-components/Notification.md

```diff
@@ -42,7 +48,27 @@ To create a notification using property, follow these steps:
     notification.Post(3000);
     ```
 
-3. Respond to OnWindowTouch:
+3. Set the notification content:
+
+    ```cs
+    // Sets text in Notification

```

Comment:
Period missing and n lower case
// Sets text in notification

or

 // Sets text in notification

Reply:
@patrykka 
```suggestion 
    // Sets text in notification.